src/Containers/VendorsList/Profile.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import styled from 'styled-components';
+import { FeaturedProfile, IFeaturedProfileProps } from '../FeaturedProfile';
+import { HeaderRow, HeaderRowProps } from '../HeaderRow';
+import { MainInterface, ResponsiveInterface } from '../../Utils/BaseStyles';
+import { flex, media } from '../../Utils/Mixins';
+
+export interface IProfileProps extends MainInterface, ResponsiveInterface,React.HTMLAttributes<HTMLDivElement> {
+    headerRowProps?: HeaderRowProps;
+    imageUrl?: string;
+    key: number;
+    name: string;
```

Comment:
This name and email and URL typing is repeated, why not just extend the other interface? No duplicate code.

Reply:
Would it be recommended to move interfaces to another .ts file because if I extend it, it would give me a dependency error, since I'm giving it the options to extend props to the profile component already?